src/main/java/frc/robot/commands/AimAtTarget.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Navigation;
+import frc.robot.subsystems.drive.Drive;
+
+public class AimAtTarget extends Command {
+    Drive driveSubsystem;
+    Navigation navSubsystem;
+    int targetId;
+
+    public AimAtTarget(Drive driveSubsystem, Navigation navSubsystem, int targetId) {
+        this.driveSubsystem = driveSubsystem;
+        this.navSubsystem = navSubsystem;
+        this.targetId = targetId;
+
+        addRequirements(driveSubsystem, navSubsystem);
```

Comment:
The drivetrain is required, but not used in this command. We should probably get #30 merged so you can build on that

Reply:
Yeah, I left that as I need a drive method to call with the heading error ( with a pid controller in drive? Or I could add the controller to navigation)

Reply:
The PID controller would be in drive and would be a new state (or maybe @AlexEvergarden has already made a placeholder state for this). That PR is approved so it can be merged whenever

Reply:
Ok I'll work on that tonight with Alex